faker/providers/geo/el_GR/__init__.py

```diff
@@ -0,0 +1,30 @@
+# coding=utf-8
+
+from __future__ import unicode_literals
+
+from decimal import Decimal
+
+from .. import Provider as GeoProvider
+
+
+class Provider(GeoProvider):
+
+    poly = (
+        (40.34026, 19.15120),
+        (42.21670, 26.13934),
+        (35.55680, 29.38280),
+        (34.15370, 22.58810),
+    )
+
+    def local_latlng(self):
+        return float(self.local_latitude()), float(self.local_longitude())
```

Comment:
I'm thinking we could move this to the superclass in `faker/providers/geo/__init__.py`. I think it would be the same implementation for most localized providers.

Reply:
Thinking through this and experimenting, I'm not sure this can work. `local_latlng` depends on the presence of `local_latitude` and `local_longitude`. But what does "local" mean in the context of a non-localized provider? The only option would be to have them provide random coords, with a docstring suggesting that you implement a localized override for your locale. So I could make it work, but it would be meaningless, since "local_foo" wouldn't be local at all (unless we do IP detection and lookup!)

The only option I can think of that would really work would be to make it take a required argument `country_code=XY` and then query the land_coords list for tuples matching that country code. Not sure how performant that would be, but it could work. Let me know if you want me tackle this (or if you want me to make local_* return random coords if not overridden).

Reply:
Querying the land_coords list for a global local_latlng with required country code seems to be very fast, and not a  performance concern. I've pushed another commit implementing this idea, but can undo it if you don't approve.

The odd thing about this is that it depends on passing two-letter country codes ('US'), rather than five-character ('en_US') locales used elsewhere in Faker. Is this a problem? It makes sense because we're interested here in a country, not a language.

Reply:
I think it makes sense for most countries, but not necessarily for all. Some countries have administrative subdivisions with different official languages. Switzerland's Cantons and Canada's Provinces come to mind.

I don't think we need to implement those right away. I'd rather leave it to people living in those countries, they know what's more appropriate. But I think we need to make it possible to support that use case.

Reply:
> with different official languages

To be clear, this feature is language-independent. You pass in a two-letter country code (not a five-char language identifier) and you get back a pair of coordinates in that country. e.g. `fake.local_latlng('AU')`.  And of course indpendent locales can override the implementation.

That said, do you accept the approach of the last commit? And if so, is there anything else for me to do here?

Reply:
I think this is good enough for now :) Thank you so much for all the hard work!